src/utils/utils.npm.ts

```diff
@@ -187,3 +187,20 @@ export function updateNpmScripts() {
 
   writePackageJson(packageJson);
 }
+
+export function cleanUpPackageJson() {
+  const packageJson = getPackageJson();
+  packageJson.scripts = sortKeysAlphabetically(packageJson.scripts);
+  packageJson.dependencies = sortKeysAlphabetically(packageJson.dependencies);
+  packageJson.devDependencies = sortKeysAlphabetically(packageJson.devDependencies);
+  writePackageJson(packageJson);
+}
+
+function sortKeysAlphabetically(obj: Record<string, string>) {
```

Comment:
Maybe use a lib (`ramda` / `lodash` / whatever) instead of writing custom util functions like this?

Reply:
I'm not a fan of introducing full dependencies for single simple functions. Let's keep it like this for now.